infra/testApi.mjs

```diff
@@ -50,5 +50,10 @@ const sample3 = (await (await fetch('http://127.0.0.1:8000/api/sample', { method
 
 const localerror = (await (await fetch('http://127.0.0.1:8000/api/localerror', { method: 'POST', body: JSON.stringify({ formula: '(FPCore (x) (- (sqrt (+ x 1)) (sqrt x)))', sample: sample2.points }) })).json())
 
+
+const explain = (await (await fetch('http://127.0.0.1:8000/api/explanations', { method: 'POST', body: JSON.stringify({ formula: '(FPCore (x) (- (sqrt (+ x 1)) (sqrt x)))', sample: sample2.points }) })).json())
+
+console.log(explain)
```

Comment:
Can we test anything specific about `explain` instead of just logging it? This doesn't have to be super complex, but even, like, checking that it's an array of length 1 or more? And perhaps that the first element has an operation of `-`?

Reply:
And let's not log—it's typically best if unit tests are silent.